wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -270,24 +270,63 @@ A notice should be placed where appropriate in a section, but must start off the
 - `Caution` should be used to warn the reader to avoid unintended consequences.
 - `Warning` should be used to warn the reader that action may be taken against them.
 
+## Stacked notices and hatnotes
+
+Multiple notices and hatnotes may be stacked when necessary. When doing this, they must be stacked without blank lines and use trailing backslashes:
+
+```markdown
+*Warning: {warning}.*\
+*See also: {article}*
+```
+
+In many cases, it may be more fitting to embed extraneous notices or hatnotes into paragraph text instead of stacking many of them.
+
 ## Emphasising
 
-### Bold
+### Bolding
 
-Bold must use double asterisks (`**`).
+Bold text must use double asterisks (`**`).
 
 Lead paragraphs may bold the first occurrence of the article's title.
 
 ### Italics
 
 Italics must use single asterisks (`*`).
 
-Names of work or video games should be italicised. osu!—the game—is exempt from this.
-
 The first occurrence of an abbreviation, acronym, or initialism may be italicised.
 
 Italics may also be used to provide emphasis or help with readability.
 
+Names of work or video games should be italicised. osu!—the game—is exempt from this.
+
+As an example, when referring to songs in the format of `{artist} - {title}`, the whole part is a reference to the work and should therefore be italicised:
+
+```markdown
+*cYsmix - triangles* is a one of the three intro songs that can be heard when starting the game client.
+```
+
+Artist names are otherwise generally not italicised. This means that in free-form references, only the title should be italicised, because the artist name is then not part of the name of the work:
+
+```markdown
+*Blue Zenith* by xi is an infamous song in the osu! community due to a famous score set by a top player on a beatmap of the song.
+```
+
+### Emphasis and links
+
+Linked text appears in a different colour which already provides emphasis, and therefore does not need further emphasis:
+
+```markdown
+[Camellia - OOPARTS] is an example of a song officially created specifically for osu!, otherwise known as an *osu! original*, since it was specifically commissioned as the osu! World Cup 2020 tiebreaker song.
```

Comment:
ideally i'd link to the track directly on the fa listing, but it seems like it's only possible to link to the artist. (i must've realised this and put it off for later, but forgotten about it.) i've used the bandcamp link instead found in the [yt video](https://www.youtube.com/watch?v=lc4nTM6M9KY) description

```suggestion
[Camellia - OOPARTS](https://cametek.bandcamp.com/track/parts) is an example of a song officially created specifically for osu!, otherwise known as an *osu! original*, since it was specifically commissioned for the [osu! World Cup 2020 tiebreaker](https://osu.ppy.sh/beatmapsets/1312143#osu/2719462).
```

Reply:
or rather, there should only be a link on the part that's being exemplified

```suggestion
[Camellia - OOPARTS](https://cametek.bandcamp.com/track/parts) is an example of a song officially created specifically for osu!, otherwise known as an *osu! original*, since it was specifically commissioned for the osu! World Cup 2020 tiebreaker.
```

Reply:
the actual link doesn't matter, it's just that its presence makes the example more clear (as in, which part is not italicized)

Reply:
yes i agree i completely forgot to link the song (i even put the brackets in <.<)